src/main/java/com/ethioclicks/servlet/ViewStudent.java

```diff
@@ -0,0 +1,62 @@
+package com.ethioclicks.servlet;
+
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import org.hibernate.cfg.Configuration;
+
+import com.ethioclicks.entity.StudentInfo;
+
+public class ViewStudent extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+
+	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+	
```

Comment:
apply the feedbacks given above, naming, formatting, using function to create session and error handling